### ISSUE TITLE
Ensure files created by response.php are unique

### ISSUE THREAD

**Issue by jmfee-usgs** [usgs/earthquake-dyfi-response#16]
In case there are multiple instances sharing the same output directory

**Comment by vinceq-usgs** [comment]
Just to clarify. This means unique across all containers right? They need to be unique when they get to backend, since (ideally) backend will only see a filename, not know which container they come from.

**Comment by jmfee-usgs** [comment]
That's correct.  Each container has a unique id, which will be part of the file name.  We weren't sure whether you read anything out of the filename, and whether we could alter the name format...

**Comment by vinceq-usgs** [comment]
No I don't do anything with the filename so you can do anything with it.
